rj_gameplay/rj_gameplay/tactic/striker_tactic.py

```diff
@@ -0,0 +1,80 @@
+from dataclasses import dataclass
+from typing import List, Optional
+import stp.rc as rc
+import stp.tactic as tactic
+import stp.role as role
+from rj_gameplay.skill import shoot, capture #receive
+import stp.skill as skill
+import numpy as np
+
+
+class shoot_cost(role.CostFn):
+    def __call__(self, robot: rc.Robot, prev_result: Optional["RoleResult"], world_state: rc.WorldState) -> float:
+
+    	#TODO: elaborate cost function:
+    	if robot.id == 1:
+    		return 0
+    	else:
+    		return 999
+
+    	# if -world_state.field.penalty_long_dist_m / 2 - 0.5 < robot.pose[0] < \
+    	# 	world_state.field.penalty_long_dist_m / 2 + 0.5 and \
+    	# 	world_state.field.their_goal_loc - world_state.field.penalty_short_dist_m - 0.5 < \
+    	# 	robot.pose[1] < world_state.field.their_goal_loc[1] - 0.2:
+    	# 	return 0
+    	# else:
+    	# 	return 999
+
+
+class StrikerTactic(tactic.ITactic):
+	"""
+	A striker tactic which captures then shoots the ball
+	"""
+
+	def __init__(self):
+		self.capture = tactic.SkillEntry(capture.Capture()) 
+		#TODO: add receive skill 
+		self.cost = shoot_cost()
```

Comment:
Add a param for a cost function in the init, so we can put in any one we want in it.